### PR TITLE
Integrate fee RPC

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

**Comment by kianenigma** [comment]
Reverted back and if the CI is green, should be ready to go

I think we've **never** tested the RPC in action though.

**Comment by kianenigma** [comment]
except for bumping versions. Are we doing that here too?

**Comment by gavofyork** [comment]
no need to bump versions for now - i already bumped to 1008 in a previous PR.

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -379,6 +380,9 @@ impl collective::Trait<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const CandidacyBond: Balance = 100 * DOLLARS;
 	pub const VotingBond: Balance = 5 * DOLLARS;
+	pub const TermDuration: BlockNumber = 10 * MINUTES;
+	pub const DesiredMembers: u32 = 13;
+	pub const DesiredRunnersUp: u32 = 0;
```
Comment (gavofyork) [inline runtime/src/lib.rs]:
```suggestion
	pub const DesiredRunnersUp: u32 = 7;
```

**Pull request merged**